python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -81,6 +104,8 @@ def set_sorted(
         -------
         Self with metadata set.
         """
+        if self.obj.size() == 1:
```

Comment:
length-1 columns are always sorted. Perhaps?

```suggestion
        if self.obj.size() <= 1:
```